### PR TITLE
chore: Deprecate BB10 MCUs

### PR DESCRIPTION
**Pull request by stylesuxx**
Build: https://github.com/bird-sanctuary/bluejay/suites/10953401347/artifacts/554378763

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,15 @@ Bluejay aims to be an open source successor to BLHeli_S adding several improveme
 
 See the project [changelog](CHANGELOG.md) for a list of changes.
 
+## Supported Platforms
+- SiLabs EFM8 BB21
+- SiLabs EFM8 BB51
+
+### Deprecated Platforms
+The `SiLabs EFM8 BB10` has been deprecated. Thas last official release is [0.18.1](https://github.com/bird-sanctuary/bluejay/releases/tag/v0.18.1). The BB10 has it's limitations in comparison to the BB21 and BB51. Not many of those seem to be around anymore and the team does not have a single one for testing, thus decided to drop support.
```
Comment (damosvil) [inline README.md:33]: Thas last official release... -> The last official release...

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Bluejay aims to be an open source successor to BLHeli_S adding several improveme
 ## Current Features
 
 - Digital signal protocol: DShot 150, 300 and 600
-- Bidirectional DShot: RPM telemetry
+- Bidirectional DShot: RPM telemetry and [extended DSHOT telemetry](https://github.com/bird-sanctuary/extended-dshot-telemetry)
```
Comment (damosvil) [inline README.md:17]: For BB21 snd BB51 speeds are now 300, and 600, for DSHOT, BB51 supported 150, and 300

File: README.md (hunk 1 above)
Comment (stylesuxx) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (stylesuxx) [reply README.md:17]: Fixed.

File: README.md [hunk 3]
```diff
@@ -54,3 +63,5 @@ If you have fixed an issue or added a functionality, please feel free to submit
 
 ### Build
 Please see the [wiki](https://github.com/bird-sanctuary/bluejay/wiki/Building-from-source) for instructions on how to build Bluejay from source.
+
+TEST
```
Comment (damosvil) [inline README.md:67]: Has this text inserted unadvertedly?

Comment (stylesuxx) [reply README.md:67]: Yes, this was just to trigger the build and make sure the commenter works ;-) forgot to revert it...

**Pull request merged**